### PR TITLE
CSS: Migrate my-sites/importer/style

### LINKED ISSUE #33675: CSS: Migrate my-sites/importer/style
Migrate my-sites/importer/style

### PR DESCRIPTION
**Pull request by Stojdza**
#### Changes proposed in this Pull Request

* Migrate `my-sites/importer/style`

#### Testing instructions

* TBD

Fixes #33675

Refs #27515

### CODE REVIEW

**Comment by Stojdza** [comment]
@blowery @jsnajdr I went with one `style.scss` file per folder. Let me know is this the right approach or I should create style file for every jsx file.

File: client/my-sites/importer/author-mapping-pane.jsx [hunk 1]
```diff
@@ -19,6 +19,11 @@ import ImporterActionButtonContainer from 'my-sites/importer/importer-action-but
 import ImporterActionButton from 'my-sites/importer/importer-action-buttons/action-button';
 import ImporterCloseButton from 'my-sites/importer/importer-action-buttons/close-button';
 
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
```
Comment (jsnajdr) [inline client/my-sites/importer/author-mapping-pane.jsx]: We usually don't import the same style from multiple components: it's better to create a separate stylesheets `author-mapping-item.scss` and `author-mapping-pane.scss` and move the relevant styles there. That makes for a better separation of components.

Can you do that for the importer stylesheet?

File: client/my-sites/importer/style.scss [hunk 2]
```diff
@@ -1,335 +1,3 @@
...
-	flex-direction: column;
-	margin-right: 10px;
-}
-
-.site-importer__source-url {
-	margin-top: 4px;
-	margin-bottom: 8px;
-}
-
 .site-importer__site-importer-confirm-site-pane-container-text {
```
Comment (Stojdza) [inline client/my-sites/importer/style.scss]: @jsnajdr I'm not really sure what to do with this one, so I left it here for now. It's not in use. Should I leave it here or remove it? Or move it, but where?

**Comment by Stojdza** [comment]
@jsnajdr Done. Let me know what do you think. 

Can you also take a look at `/importer/logos/` folder? Is there a need for `.scss` file there?

File: client/my-sites/importer/style.scss (hunk 2 above)
Comment (jsnajdr) [reply client/my-sites/importer/style.scss]: It's indeed not used, so let's remove it.

**Review by jsnajdr — APPROVED** [review]
Hi @Stojdza, this looks good :+1:

The `site-importer-confirm-site-pane-container-text` style can be removed.

The `importer__shell` style is duplicated in two files, which is a bit ☹️, but harmless.

The service icon styles are 100% :+1:

**Comment by Stojdza** [comment]
Thanks @jsnajdr! I removed `importer/style.scss`. 

> The importer__shell style is duplicated in two files, which is a bit ☹️, but harmless.

Yeah, I noticed it. Is there any better solution here? I was thinking to import from the same `.scss` file, despite the fact that `site-importer/index.jsx` and `file-importer.jsx` are not in the same directory, but as you mentioned earlier -  we don't import the same style from multiple components.

**Comment by jsnajdr** [comment]
> Yeah, I noticed it. Is there any better solution here?

`importer__shell` is a `Card` with a few extra styles. There are two ways how to modularize that well:
1. Create two classnames (`importer__file-importer-card` and `importer__site-importer-card`) and duplicate the content of the rules.
2. Create a common `ImporterShellCard` component that is just a wrapper around `Card` and use it at both places:
```js
function ImporterShellCard( { className, ...props } ) {
  return <Card className={ classnames( 'importer__shell-card', className ) } { ...props } />
}
```

**Comment by Stojdza** [comment]
@jsnajdr I went with first one. Let me know is it good to go.

**Comment by jsnajdr** [comment]
Yes, ready to :ship::+1:

**Comment by Stojdza** [comment]
@jsnajdr I changed selectors and forgot to run e2e tests - they broke import tests 😬 It's fixed in [this PR ](https://github.com/Automattic/wp-calypso/pull/33881)

**Pull request merged**